backend/src/test/java/tqs/electro/electro/integrationTests/StationControllerIT.java

```diff
@@ -76,10 +77,12 @@ void whenGetAllStations_thenReturnList() throws Exception {
         );
 
         // when
-        mockMvc.perform(get("/station"))
+        MvcResult result = mockMvc.perform(get("/station"))
```

Comment:
```suggestion
        MvcResult result = mockMvc.perform(get("/backend/station"))
```

change this so the test pass on all get and post requests